game-core/src/main/java/games/strategy/triplea/delegate/remote/IAbstractPlaceDelegate.java

```diff
@@ -19,7 +19,6 @@
   @RemoteActionCode(13)
   String placeUnits(Collection<Unit> units, Territory at, BidMode bidMode);
 
-  @RemoteActionCode(12)
```

Comment:
Technically not needed, this method is only ever invoked in test code, so the annotation is redundant

Reply:
Would it be a good idea to add a comment or test annotation to this?

Reply:
Comment would be a good idea.